llvm/lib/Target/RISCV/RISCVInstrFormats.td

```diff
@@ -514,6 +514,78 @@ class RVInstJ<RISCVOpcode opcode, dag outs, dag ins, string opcodestr,
   let Inst{6-0} = opcode.Value;
 }
 
+//===----------------------------------------------------------------------===//
```

Comment:
this should be in a MIPS vendor file

Reply:
Yes, thanks, addressed in https://github.com/llvm/llvm-project/pull/121394